python/rikai/types/video.py

```diff
@@ -59,42 +58,53 @@ def __init__(self, vid: str):
     def __repr__(self) -> str:
         return "YouTubeVideo({0})".format(self.vid)
 
-    def _repr_html_(self):
-        from IPython.lib.display import YouTubeVideo
+    def show(self, **kwargs):
+        """
+        Visualization in jupyter notebook with custom options
+
+        Parameters
```

Comment:
do you have an example about how to use it?  

Are these per-object level options? Can/should we config them globally?

Reply:
these are the kwargs for the IPython.display.YouTubeVideo/Video/Image classes. I can add links to their docs.